conda/environment.yml

```diff
@@ -20,22 +20,18 @@ variables:
 
 dependencies:
   - python=3.8
-  - pythonocc-core=7.4.1
   - freecad=0.19.2
-  - freecad.python=0.0.3
-  - gmsh=4.6.0
-  - python-gmsh=4.6.0
-  - meshio=4.4.5
-  - mshr=2019.1.0
+  - python-gmsh=4.8.4
+  - gmsh=4.8.4
```

Comment:
Might want to check with @ivanmaione if this upgrade is OK, as we're still developing the meshing and I don't think we have a test suite for it yet.

Reply:
The changes do not seem to be that major according to the gmsh changelog. It looks like they are all optional additional arguments

Reply:
There aren't meshing tests yet, but there is at least a tutorial - I can update my environment and give that a run at least... if there are differences then given that the meshing isn't used yet then I'd suggest it's best to upgrade and fix things up after the upgrade.

Reply:
In that case gmsh was updated over the weekend so I've pushed the most recent version